File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

// Convert takes in a number and turns it into rain language
func Convert(number int) string {
	strings := []string{
		"Pling",
		"Plang",
		"Plong",
	}
	var rain string
	if number%3 == 0 {
		rain += strings[0]
	}
	if number%5 == 0 {
		rain += strings[1]
	}
	if number%7 == 0 {
		rain += strings[2]
	}
	if rain != "" {
		return rain
	}
	rain = strconv.Itoa(number)
	return rain
}

File: solutions/python/raindrops/1/raindrops.py
def convert(number: int) -> str:
    """Converts a number into it's corresponding rain sound

    :param number: int - The number convert
    :return: str - The rain sound that corresponds to the provided number

    Function that takes in an integer and returns a string that corresponds
    to the given integer.
    """
    sound = ""

    if number % 3 == 0:
        sound += "Pling"
    if number % 5 == 0:
        sound += "Plang"
    if number % 7 == 0:
        sound += "Plong"

    return sound if len(sound) > 0 else str(number)
